texar/evals/bleu_transformer.py

```diff
@@ -185,14 +194,18 @@ def bleu_tokenize(string: str) -> List[str]:
     return string.split()
 
 
-def bleu_wrapper(ref_filename: str,
-                 hyp_filename: str,
-                 case_sensitive: bool = False) -> float:
+def file_bleu(ref_filename: str,
+              hyp_filename: str,
+              bleu_version: str = "corpus_bleu_transformer",
+              case_sensitive: bool = False) -> float:
     r"""Compute BLEU for two files (reference and hypothesis translation).
```

Comment:
If `file_bleu` and `bleu_tokenize` are not specific for the bleu-transformer version, they can be moved to another file (either `bleu.py` or a new file)

Reply:
I prefer to keep these two functions in `bleu_transformer` for now. Currently `bleu_tokenize` is just a special case of the original implementation of the tokenization in `multi-bleu.perl` (This is also mentioned in `doc` of `bleu_tokenize`).  I rename `bleu_tokenize` to `bleu_tokenize_transformer`. After we implement the general `bleu_tokenize`, we could move /delete these two function to the correct folder.

Reply:
Ok. I guess `bleu_transformer_tokenize` could prbably a better name?